### PR TITLE
Tutorial "Pre-commit hooks"

### PR DESCRIPTION
**Pull request by seeM**

### CODE REVIEW

File: nbdev/qmd.py [hunk 1]
```diff
@@ -72,3 +72,18 @@ def tbl_sep(sizes:int|list=3  # List of column sizes, or single `int` if all siz
     "Create a markdown table separator with relative column size `sizes`"
     if isinstance(sizes,int): sizes = [3]*sizes
     return tbl_row('-'*s for s in sizes)
+
+# %% ../nbs/api/qmd.ipynb 10
+def _install_nbdev():
```
Comment (seeM) [inline nbdev/qmd.py:77]: @hamelsmu @jph00 I'd like to hear your take on this

Comment (seeM) [reply nbdev/qmd.py:77]: More generally, what do you think about moving repeated doc snippets into internal functions like this? This PR uses `_install_nbdev` in git friendly jupyter, tutorial, and pre commit doc pages.

Comment (jph00) [reply nbdev/qmd.py:77]: lgtm!

**Comment by jph00** [comment]
Many thanks for adding this helpful tutorial!

**Pull request merged**